Modules/_sqlite/module.c

```diff
@@ -244,7 +244,7 @@ load_functools_lru_cache(PyObject *module)
 static PyMethodDef module_methods[] = {
     PYSQLITE_ADAPT_METHODDEF
     PYSQLITE_COMPLETE_STATEMENT_METHODDEF
-    PYSQLITE_CONNECT_METHODDEF
+    {"connect", _PyCFunction_CAST(pysqlite_connect), METH_FASTCALL|METH_KEYWORDS, pysqlite_connect__doc__},
```

Comment:
`pysqlite_connect` is defined not by Argument Clinic, so we should not use PyMethodDef initializer generated by Argument Clinic.